tests/integration_tests/datasets/api_tests.py

```diff
@@ -660,7 +661,11 @@ def test_create_dataset_validate_uniqueness(self):
         assert rv.status_code == 422
         data = json.loads(rv.data.decode("utf-8"))
         assert data == {
-            "message": {"table_name": ["Dataset energy_usage already exists"]}
+            "message": {
+                "table": [
+                    f"Dataset {Table(energy_usage_ds.table_name, schema)} already exists"
```

Comment:
The schema differs between SQLite and PostgreSQL and thus this is parameterized.